sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/Rule.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+public class Rule {
```

Comment:
Are rules something we have in the first preview? if not, we should remove it for now. Else, this is in the public API.